### ISSUE TITLE
CMake: Unable to build BDS library

### ISSUE THREAD

**Issue by ghost** [LiteLDev/LiteLoader#805]
### Exceptions module

Core

### Operating System

Windows Server 2022

### LiteLoader version

2.6.2 Beta

### BDS version

None

### What happened?

I ran into a problem building LiteLoader via Github Action (CMake). The build hangs at the "Build Library" stage and nothing further happens. No changes were made in Liteloader Dev Library Builder.

![image](https://user-images.githubusercontent.com/44939683/190860530-d3315925-e5e3-494e-91a6-1a627ca6ab3e.png)

### Steps to reproduce?

1. Fork repository with additional branches
2. Enable Actions and run CMake workflow in beta branch

### Relevant log output

```shell
/////////////////////////////////////////////////////
  //                                                 //
  //         LiteLoader Dev Library Builder          //
  //                                                 //
  /////////////////////////////////////////////////////
---Start Processing...
---- Loading bedrock_server.pdb ...
---- Generating Libs...
```

**Comment by Jaciezyt** [comment]
What is your command?
Please pass -o parameter, and see https://github.com/LiteLDev/PermissionAPI/tree/main/.github/workflows/build.yml
I once met this problem, too

**Comment by ghost** [comment]
> What is your command?

https://github.com/LiteLDev/LiteLoaderBDS/blob/e32cfb19a3ef9168856211ed53e5a1abd29aebe5/.github/workflows/cmake.yml#L56-L57

> Please pass -o parameter

Nothing changed, also not building

![image](https://user-images.githubusercontent.com/44939683/190861425-fc208244-81e9-4e2b-a81f-241fa28a161d.png)

**Comment by Jaciezyt** [comment]
@shishkevichd try `LibraryBuilder.exe  -o ../Lib/ ./Server/`

**Comment by RimuruChan** [comment]
This may be a bug. needs further confirmation.
We did not find it due to caching of libs.

**Comment by RimuruChan** [comment]
Can reproduce.
The error is caused by a non-existent Libs directory causing LibraryBuilder to pop up an error window blocking the workflow.

**Comment by RimuruChan** [comment]
Fixed in #806
